task.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+# создаём каталог task с вложенными директориями
+# task
+#   dir1
+#   dir2
+#   dir3
+#       dir4
+mkdir /task
+mkdir ./dir1 ./dir2 ./dir3
```

Comment:
`./` можно не ставить везде
```suggestion
mkdir dir1 dir2 dir3
```